pkg/agent/interfacestore/interface_cache.go

```diff
@@ -190,11 +190,15 @@ func (c *interfaceCache) GetContainerInterface(containerID string) (*InterfaceCo
 	return objs[0].(*InterfaceConfig), true
 }
 
-// GetContainerInterfacesByPod retrieves InterfaceConfigs for the Pod.
+func (c *interfaceCache) GetContainerInterfacesByEntity(name, namespace string) []*InterfaceConfig {
```

Comment:
I would not call it GetContainerInterfaces, as for external entities, it should not be "container" interfaces. Could we just name it GetInterfacesByEntity or GetInterfacesByEntityName?